src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ConfigurationTests.cs

```diff
@@ -339,9 +340,9 @@ private string CleanWhitespace(string rawText)
             return string.Concat(rawText.Where(c => !char.IsWhiteSpace(c)));
         }
 
-        private string ConstructSettingsJson(params string[] permittedFileNames)
+        internal static string ConstructSettingsJson(string sampleConfigurationsDirectory, params string[] permittedFileNames)
```

Comment:
The changes related to adding this parameter seem like they can be undone.

Reply:
This is also being used in `TemplatesTests`; I originally had just included the template testing directly in this file, but decided it made sense to separate it as its own thing. If we don't like having another test file accessing this method within `ConfigurationTests`, I could break it out into a helper class - I wasn't sure if that was considered necessary, but given that it wasn't obvious that this is being used elsewhere, I suspect that I should.